apigentools/__main__.py

```diff
@@ -0,0 +1,3 @@
+from apigentools.cli import cli
```

Comment:
```suggestion
# Unless explicitly stated otherwise all files in this repository are licensed
# under the 3-clause BSD style license (see LICENSE).
# This product includes software developed at Datadog (https://www.datadoghq.com/).
# Copyright 2020-Present Datadog, Inc.
from apigentools.cli import cli
```